suite.go

```diff
@@ -724,39 +670,36 @@ func parseFeatureDir(dir string) ([]*feature, error) {
 
 func parsePath(path string) ([]*feature, error) {
 	var features []*feature
-	// check if line number is specified
-	var line int
-	path, line = extractFeaturePathLine(path)
+
+	path, line := extractFeaturePathLine(path)
 
 	fi, err := os.Stat(path)
 	if err != nil {
 		return features, err
 	}
 
+	newIdFunc := (&messages.Incrementing{}).NewId
```

Comment:
Go should yell at you for this variable name. :p It should be newIDFunc instead of newIdFunc

Reply:
haha, true, will fix

would be nice to add more linting in general

Reply:
I don't think our linting is running as it should btw. it misses more things, like the colors pkg.

Reply:
Can look into that. I personally use the go extension in VS Code which lints and shows issues on its own. It’s awesome! :p